samples/src/JustSaying.Sample.Restaurant.KitchenConsole/appsettings.json

```diff
@@ -1,11 +1,4 @@
 {
   "AWSRegion": "eu-west-1",
-  "AWSServiceUrl": "http://localhost:4100",
```

Comment:
Only meant for this to be empty string, as then it attempts to connect to real SQS instead of localstack. Will fix 👍 .

Reply:
Hmm, this diff is a bit confusing as it doesn't show that that line is now:

` "AWSServiceUrl": "",`

If you meant to question that, it's because this sample didn't work with my localstack, and I wanted to test it against real SQS. Can you think of a way we can leave this setting while devving against real SQS?